docs/audio.rst

```diff
@@ -41,9 +70,25 @@ Classes
 Using audio
 ===========
 
-You will need a sound source, as input to the ``play`` function. You can generate your own, like in
-``examples/waveforms.py``.
+You will need a sound source, as input to the ``play`` function. You can use
+the built-in sounds **V2** from the ``microbit`` module, ``microbit.Sound``, or
+generate your own, like in ``examples/waveforms.py``.
+
+Built-in sounds **V2**
+----------------------
 
+The built-in sounds can be called using ``audio.play(Sound.NAME)``.
+
+    * ``Sound.GIGGLE``
+    * ``Sound.HAPPY``
+    * ``Sound.HELLO``
+    * ``Sound.MYSTERIOUS``
+    * ``Sound.SAD``
+    * ``Sound.SLIDE``
+    * ``Sound.SOARING``
+    * ``Sound.SPRING``
+    * ``Sound.TWINKLE``
+    * ``Sound.YAWN``
 
 Technical Details
 =================
```

Comment:
I would change `The ``audio`` module consumes samples at 7812.5 Hz,...` to `The ``audio`` module consumes ``AudioFrame`` samples at 7812.5 Hz,...`

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/01331f3e9d0e8ea461e9fae271fe3eb62153802f